#sensor map data
#
#ROSヘッダ
#std_msgs/Header header
geometry_msgs/Point32[] data #セル内の点群の重心
std_msgs/UInt32[] index